### ISSUE TITLE
Heading structure and title are wrong

### ISSUE THREAD

**Issue by tombye** [alphagov/notifications-tech-docs#101]
The docs for each language are a single page but have two `<h1>`s. Also, the `<title>` for each set of docs doesn't mention which API client those docs are for.

Both these are likely to fail an accessibility audit due to their effect on users of assistive technologies.

## Multiple H1s

This section in the HTML5 spec explains that no assistive technologies support document outlines with multiple h1s: 

https://www.w3.org/TR/html52/sections.html#creating-an-outline

![notify_tech_docs_multiple_h1s](https://user-images.githubusercontent.com/87140/69722349-c0cb8880-110e-11ea-8848-c046e98f7810.png)

This means the structure of these docs is currently broken.

This does help you orient yourself when reading the links in the nav.

The tech-docs-gem only adds a bold link for a `<h1>`s. If we didn't use `<h1>`s for the section headings, the nav would look like that in the WCAG Primer:

![wcag_primer](https://user-images.githubusercontent.com/87140/69722665-9e863a80-110f-11ea-906a-78d1fc92a6c9.png)

## Generic text in the title element

The `<title>` element for the docs for each language is the same:

"Client documentation - GOV.UK Notify | Notify"

This matches an example of how to fail success criteria 2.4.2 in the WCAG docs:

https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/F25#F25-failex2

**Comment by jonathanglassman** [comment]
Hi @tombye 

Thanks for this. I agree that this should be changed. There are 2 ways we can do this.

1) Add on a level to the headings
2) Keep headings as is but change the docs from single page to multipage. 

Obviously option 1 is easier, but if we make the docs multipage and enable the search box, it might increase docs usability. We haven't had any negative feedback on moving other docs from single to multipage (e.g. PaaS, Pay). @quis @servingUpAces what do you think?

**Comment by quis** [comment]
If I remember right there was something blocking us adopting multipage and search for our docs. I’m all in favour of using those features if they’ve been shown to work for other teams. 

@tombye maybe you could spike multipage docs and see if problems still exist?

**Comment by klssmith** [comment]
I think one of the issues before with switching the site to multi-page and enabling search was that if you search for how to send an email (for example) you will see results from all 6 client docs, when you probably only want to see the results for the client you're interested in. There may be a way round that though - I haven't tried it out recently

**Comment by tombye** [comment]
What about if we add another heading level, like Jon said, and re-style the table-of-contents so it keeps the bold style for `<h2>`s and makes the `<h1>` the next text size up?

Like:

<img width="542" alt="image" src="https://user-images.githubusercontent.com/87140/69731577-0c3b6200-1122-11ea-9f17-e7024f32a471.png">

**Comment by tombye** [comment]
> What about if we add another heading level, like Jon said, and re-style the table-of-contents so it keeps the bold style for `<h2>`s and makes the `<h1>` the next text size up?
> 
> Like:
> 
> <img alt="image" width="542" src="https://user-images.githubusercontent.com/87140/69731577-0c3b6200-1122-11ea-9f17-e7024f32a471.png">
> 
> Note: I only mean for our docs. It wouldn't apply these styles upstream to the tech_docs_gem.

Discussing this with @quis, we reckon it's worth doing.

**Comment by jonathanglassman** [comment]
Best practice for the tech writer community is to minimise bespoke implementations of the tech doc template. Is there any way we can do this without tweaking the css?

**Comment by tombye** [comment]
Sorry @jonathanglassman, you're right of course :)

I looked at all the docs sites I could find that also had a single-page:
1. https://docs.publishing.service.gov.uk/
2. https://content-api.publishing.service.gov.uk/#gov-uk-content-api
3. https://gds-way.cloudapps.digital/
4. https://alphagov.github.io/wcag-primer/

Number 1. seems to have a table of contents like the one we have, visually at least, but with the right heading structure. It does this by not including a link to the h1 and having all the links to h2s as bold. I'm gonna look into this as an alternative.

**Comment by klssmith** [comment]
The issue of generic text in the title element is now fixed - https://github.com/alphagov/notifications-tech-docs/pull/111. Multiple H1s still needs to be looked at.

**Comment by quis** [comment]
This was fixed by https://github.com/alphagov/notifications-tech-docs/pull/136
